Makefile

```diff
@@ -232,3 +243,7 @@ $(SCALITY_EL7_REPODATA): $(SCALITY_EL7_RPMS) | $(PACKAGE_BUILD_CONTAINER)
 		--rm \
 		$(PACKAGE_BUILD_IMAGE) \
 		/entrypoint.sh buildrepo
+
+$(ISO_ROOT)/images:
```

Comment:
I would put as dependency for the target all the files here
Right now the target won't be executed a second time if the directory exists and isn't changed.

Reply:
done